docs/tutorials/training_to_inference.md

```diff
@@ -0,0 +1,201 @@
+├── efficientnet-b0
+│   ├── 1
+│   │   └── model.plan
+│   └── config.pbtxt
+├── efficientnet_ensemble
+│   ├── 1
+│   └── config.pbtxt
+└── preprocessing
+    ├── 1
+    │   └── dali.py
+    └── config.pbtxt
+```
+
+Last step is the configuration of DALI model in the model repository. As mentioned earlier, you can use Triton's
+Model Autoconfiguration or provide explicit `config.pbtxt` file. According to the outline above, below is the example of
+simple configuration file for DALI model (`model_repository/preprocessing/config.pbtxt`). Please note that the name of
+the `input` matches the name of the `fn.external_source` operator provided in the `inference_pipe`.
+
+```
+name: "preprocessing"
+backend: "dali"
+max_batch_size: 32
+input [
+  {
+    name: "DALI_INPUT"
+    data_type: TYPE_UINT8
+    dims: [ -1 ]
+  }
+]
+
+output [
+  {
+    name: "DALI_OUTPUT"
+    data_type: TYPE_FP32
+    dims: [ 3, 224, 224 ]
+  }
+]
+```
+
+For the details about the model configuration files please refer to [Triton documentation](https://docs.nvidia.com/deeplearning/triton-inference-server/user-guide/docs/user_guide/model_configuration.html).
+
+### ➔ 3 : Run Triton server and send requests
+
+Please refer to [Triton documentation](https://docs.nvidia.com/deeplearning/triton-inference-server/user-guide/docs/index.html)
+for any details about running the server and leveraging Triton Client module to send requests to the server.
+
+### Triton Client hints
+
+As mentioned earlier, you can find the examples of Triton client implementation inside [DALI Backend repository](https://github.com/triton-inference-server/dali_backend/tree/main/qa). 
+The purpose of this tutorial is not to fully explain how to create a Triton client module. However, the following are 
+a couple of suggestions for creating a Triton client specifically for DALI model.
+
+#### Loading data as binary buffer
+NVIDIA hardware offers image and video decoding acceleration using [Hardware JPEG Decoder](https://developer.nvidia.com/blog/leveraging-hardware-jpeg-decoder-and-nvjpeg-on-a100/) and [NVDEC cores](https://developer.nvidia.com/video-codec-sdk). 
+Since DALI provides easy-access Python interface for both, you'd typically like to perform data decoding on the server side. 
+Therefore, the Triton client will typically send encoded data to the server. We suggest using `np.fromfile` or similar 
+approach to read the binary data from disk (this one prooved to be the fastest):
+
+```python
+def load_image(image_path):
+    return np.fromfile(image_path, dtype=np.uint8)
+```
+
+#### Batching data in the request
+
+With Triton and DALI Backend you can leverage batching in two ways. Firstly, you can turn on [Dynamic Batching](https://docs.nvidia.com/deeplearning/triton-inference-server/user-guide/docs/user_guide/model_configuration.html#dynamic-batcher). 
+With this feature, you send the samples one-by-one to the server and the server automatically puts together a batch of data for you. 
```

Comment:
`one-by-one` or any lenght?

Reply:
In theory any length, but the most common usage is that the samples are sent one by one. Removed the `one-by-one`.